app/containers/InterventionChannelProvider/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const INTERVENTION_CHANNEL_ROUTES = [
+  '/interventions/:interventionId',
+  '/interventions/:interventionId/sessions/:sessionId/edit',
+  '/interventions/:interventionId/sessions/:sessionId/settings',
+  '/interventions/:interventionId/sessions/:sessionId/report-templates',
+  '/interventions/:interventionId/sessions/:sessionId/generated-reports',
+  '/interventions/:interventionId/sessions/:sessionId/sms-messaging',
+  '/interventions/:interventionId/sessions/:sessionId/map',
+];
```

Comment:
const baseInterventionRoute = '/interventions/:interventionId';
const specificInterventionRoute = `${baseInterventionRoute}/sessions/:sessionId/`
```suggestion
export const INTERVENTION_CHANNEL_ROUTES = [
   baseInterventionRoute,
  `${specificInterventionRoute}/edit`,
  `${specificInterventionRoute}/settings`,
  `${specificInterventionRoute}/report-templates`,
  `${specificInterventionRoute}/sms-messaging`,
  `${specificInterventionRoute}/generated-reports`,
  `${specificInterventionRoute}/map`,
];
```